ui-admin/src/portal/siteContent/SiteContentView.tsx

```diff
@@ -112,7 +140,26 @@ const SiteContentView = ({ portalEnv, portalShortcode }: {portalEnv: PortalEnvir
     submitMailingListContact: () => Promise.resolve({})
   }
 
-  return <InitializedSiteContentView siteContent={portalEnv.siteContent} previewApi={previewApi}/>
+  const loadSiteContent = async (stableId: string, version: number) => {
+    setIsLoading(true)
+    Api.getSiteContent(portalShortcode, stableId, version).then(response => {
+      setSiteContent(response)
+      setIsLoading(false)
```

Comment:
Right now this will spin forever if the API errors out. I know we show the error notification but I don't think we should give the impression that we're still attempting to load content.

Error handling is probably an area that could use some improvement and consistency across the admin tool, so this isn't blocking for this PR.

Reply:
JN-537 created